renku/cli/dataset.py

```diff
@@ -243,6 +243,18 @@ def unlink(client, name, include, exclude, yes):
         click.secho('OK', fg='green')
 
 
+@dataset.command('rm')
+@click.argument('names', nargs=-1)
+@pass_local_client(clean=True, commit=True)
+def remove(client, names):
+    """Delete a dataset."""
+    datasets = [str(client.dataset_path(name)) for name in names]
+    with progressbar(datasets) as bar:
```

Comment:
```suggestion
    with progressbar(datasets, item_show_func=lambda item: str(item) if item else '') as bar:
```

Reply:
So allocating additional stack frame for every function call is better than doing it straight away?

Reply:
1. function call <<< syscall for unlink
2. we can completely remove the progressbar

Reply:
1. Same sentinel is on remove_dataset. So that's really correct. It's the same thing.
2. Why are spinning in circle then?
3. As I said, due to sentinel it's the same thing in the end. :+1: